docs/source/data/data_catalog.md

```diff
@@ -32,21 +32,21 @@ shuttles:
 ```
 ### Dataset `type`
 
-Kedro offers a range of datasets, including CSV, Excel, Parquet, Feather, HDF5, JSON, Pickle, SQL Tables, SQL Queries, Spark DataFrames and more. They are supported with the APIs of pandas, spark, networkx, matplotlib, yaml and more.
+Kedro supports a range of connectors, for CSV files, Excel spreadsheets, Parquet files, Feather files, HDF5 files, JSON documents, pickled objects, SQL tables, SQL queries, and more. They are supported using libraries like pandas, PySpark, NetworkX, Matplotlib and more.
```

Comment:
```suggestion
Kedro supports a range of connectors, for CSV files, Excel spreadsheets, Parquet files, Feather files, HDF5 files, JSON documents, pickled objects, SQL tables, SQL queries, and more. They are supported using libraries such as pandas, PySpark, NetworkX, and Matplotlib.
```

Reply:
@stichbury For my knowledge, should there be an Oxford comma here? Wondering because I came across a lot of instances where Vale wanted to add an Oxford comma, but I assumed not using one was the British way.

Reply:
I've been adopting the Oxford comma as Vale requests it. I am not a big fan of it but it does have value in tech writing for clarity so where it's missing, and flagged as absent, it is fine to add it in.